src/parser.c

```diff
@@ -108,6 +108,20 @@ const char *parseIdent(Parser *p) {
     return ident;
 }
 
+DynamicArray(const char *) parseIdentList(Parser *p) {
+    DynamicArray(const char *) names = NULL;
+
+    const char *name = parseIdent(p);
+    ArrayPush(names, name);
+
+    while (isToken(p, TK_Comma)) {
+        nextToken();
```

Comment:
Use `matchToken(p, TK_Comma)` and ditch the `nextToken`

Reply:
Done.